### ISSUE TITLE
North/South Arrows not displayed

### ISSUE THREAD

**Issue by RCura** [Groupe-ElementR/cartography#8]
In the vignette of the current CRAN version, for exemple the [choropleth map](https://cran.r-project.org/web/packages/cartography/vignettes/cartography.html#choropleth-map), the south arrow is not displayed, although it should with the use of `south = TRUE` (see code below)

```
layoutLayer(title = "Demographic Trends", author = "cartography", 
            sources = "Eurostat, 2008", frame = TRUE, col = NA, 
            scale = NULL,coltitle = "black",
            south = TRUE) # add a south arrow
```

**Comment by rCarto** [comment]
The arrow is actually displayed but is colored in a very light grey. We could add some parameters to custom the arrow.

**Comment by rCarto** [comment]
fix by c6aca7ededf42017f3f755ab6072bbe43ae4ecdf commit.
